kogito-timer-quarkus/README.md

```diff
@@ -122,11 +208,18 @@ curl -H 'Content-Type:application/json' -H 'Accept:application/json' http://loca
 ### Cancel boundary timer instance 
 
 ```
-curl -H 'Content-Type:application/json' -H 'Accept:application/json' 'http://localhost:8080/timersOnTask/{uuid}'
+curl -X DELETE 'http://localhost:8080/timersOnTask/{uuid}'
 ```
 
 where `{uuid}` is the id of the given timer instance
 
+After executing the above commands you should see a log similar to
+
+* Curl Log
+<p align="center"><img src="docs/images/curlLog.png"></p>
+
+* Quarkus Log
+<p align="center"><img src="docs/images/quarkusLog.png"></p>
```

Comment:
Can we put this as text ? It would be more readable

Reply:
fixed @radtriste